internal/infrastructure/storage/db/pg/sqlc/query.sql

```diff
@@ -0,0 +1,137 @@
+/** Market **/
+
+-- name: InsertMarket :one
+INSERT INTO market (name,base_asset,quote_asset,base_asset_precision,
+quote_asset_precision,tradable,strategy_type,base_price,quote_price) VALUES
+($1, $2, $3, $4, $5, $6, $7, $8, $9) RETURNING *;
+
+-- name: GetMarketByName :many
+SELECT * from market m inner join fee f on m.name = f.fk_market_name where name = $1;
+
+-- name: GetMarketByBaseAndQuoteAsset :many
+SELECT * from market m inner join fee f on m.name = f.fk_market_name where
+base_asset = $1 and quote_asset = $2;
+
+-- name: GetTradableMarkets :many
+SELECT * FROM market m inner join fee f on m.name = f.fk_market_name
+where tradable = true;
+
+-- name: GetAllMarkets :many
+SELECT * FROM market m inner join fee f on m.name = f.fk_market_name;
+
+-- name: UpdateMarket :one
+UPDATE market SET base_asset_precision = $1, quote_asset_precision = $2,
+tradable = $3, strategy_type = $4, base_price = $5, quote_price = $6 WHERE
+name = $7 RETURNING *;
+
+-- name: DeleteMarket :exec
+DELETE FROM market WHERE name = $1;
```

Comment:
Are the related fees deleted as well this way?

Reply:
not with that only, but yes in repo method, check in [here](https://github.com/sekulicd/tdex-daemon/blob/bbdc7ccfe53efd7c998d54a24bd8db2f3ac5da26/internal/infrastructure/storage/db/pg/market_memory_repository_impl.go#L289)